### ISSUE TITLE
Post line comments fail to parse

### ISSUE THREAD

**Issue by sbeckeriv** [pest-parser/pest-fmt#3]
Dearest Maintainer 

I checked out the repo and added my pest file to the tests. It was reaching unreachable code. After adding COMMENT to correct locations it was clear that the post line comments caused the issue. This for example works fine for pest. 
```
p = { ((SPACE_SEPARATOR |SEPARATOR | MARK | SYMBOL | NUMBER | LETTER | ":" | "_" | ":" | "?" | "|" | "," | "*" | "[" | "]" |"#" )~ NEWLINE*)+ }
// ok comment
value = _{ p }
call = {value ~ (arguments)*}
arguments = { "(" ~ (params //  bad comment
~ ("," ~ params )* // bad comment
//comment dont know if this was a problem.
~ ","?)? ~ ")"
}
params = {call ~ (call)*}

```
I moved my comments and it ran fine. in case someone else gets to this point it is documented. You could warn and eat them. or leave the line alone... collect the comments till after the current pair maybe?

Thank you for your time on this project. My file looks amazing now.
Dictated but not reviewed,
Becker

**Comment by GalAster** [comment]
Well, for 

```js
params = {"something"}
arguments = { "(" ~ (params //  bad comment
~ ("," ~ params )* // bad comment
//comment dont know if this was a problem.
~ ","?)? ~ ")"
}
```

there is no line break rule to deal with this situation.

What should it be formatted?

**Comment by sbeckeriv** [comment]
Hello @GalAster,

Great question. 

If you could. it would be nice to save the comments. Moving them to below the current bit. Allow the user to correct the comments.

arguments = { "(" ~ (params ~ ("," ~ params )*  ~ ","?)? ~ ")" } 
//  bad comment
// bad comment
//comment dont know if this was a problem.

Or Error. Return an error that says "Comment at line X does not with current rules. Please move comment to above or blow the identifier. 

Thanks for your time on this project.

Dictated but not reviewed.
Becker

**Comment by huacnlee** [comment]
This has fixed in #9 

```js
p = { ((SPACE_SEPARATOR | SEPARATOR | MARK | SYMBOL | NUMBER | LETTER | ":" | "_" | ":" | "?" | "|" | "," | "*" | "[" | "]" | "#") ~ NEWLINE*)+ }
// ok comment
value     = _{ p }
call      = { value ~ (arguments)* }
arguments = {
    "(" ~ (params // bad comment
   ~ ("," ~ params)* // bad comment
   // comment dont know if this was a problem.
   ~ ","?)? ~ ")"
}
params    = { call ~ (call)* }
```
